custom_components/powercalc/data/ikea/LED1738G7/model.json

```diff
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly",
```

Comment:
Could you please mention the full smart plug model used. For example "Shelly Plug S"

Reply:
Done.